docs/get-started/on-device-training.md

```diff
@@ -0,0 +1,49 @@
+---
+title: On-Device Training
+parent: Get Started
+nav_order: 12
+---
+
+# On-Device Training with ONNX Runtime
+{: .no_toc }
+`On-Device Training` refers to the process of training a model on an edge device, such as mobile phones, embedded devices, gaming consoles, web browsers, etc. This is in contrast to training a model on a server or a cloud. Training on the device
+
+- Can improve data privacy and security, especially when working with sensitive data that cannot be shared with a server or a cloud.
+- It is also useful for the task of personalization where the model needs to be trained on the user's data.
+- It can also be beneficial in scenarios where network connectivity is unreliable or limited.
+
+In addition to the above scenarios, there could be many more that warrant the need for `On-Device Training`.
+
+
+`ONNX Runtime Training` offers an easy way to efficiently train and infer a wide range of ONNX models on edge devices. The training process is divided into two phases:
+- [the offline phase](#the-offline-phase)
+- [the training phase](#the-training-phase).
+
+## The Offline Phase
+In this phase, training artifacts are prepared on a server, cloud or a desktop. These artifacts can be generated by using the `onnxruntime-training`'s [artifact generation python tools](./../api/python/learning_on_the_edge/training_artifacts.html) available in the python package.
+
+Refer to the [installation instructions](./../install/index.md#offline-phase---prepare-for-training)
+
+## The Training Phase
+Once these artifacts are generated, they can be deployed to production scenarios on edge devices. `ONNX Runtime` offers a wide range of packages in multiple language bindings.
+
+Refer to the [installation instructions](./../install/index.md#training-phase---on-device-training) for a complete list of all language bindings.
+
+Once training on the edge device is complete, an inference ready onnx model can be generated on the edge device itself. This model can then be used for inferencing with onnxruntime.
+
+[comment]: <> (Learn more from the blogs. Links to the blogs go here.)
+
+## Installation
+
+Refer to the [installation instructions](./../install/index.md#install-for-on-device-training) for details on how to install for your scenario.
+
+## Building from Source
+
+Refer to the [build instructions](./../build/training.md#build-for-on-device-training) for details on how to build for your custom scenario.
+
+[comment]: <> (Learn more from the tutorials. Links to the demo and website tutorial go here.)
+[comment]: <> (Also link to the onnxruntime-training-examples repo goes here.)
+
```

Comment:
Here create a placeholder for link to step by step tutorial.

Reply:
I think having an empty placeholder might not be very meaningful right now. It may confuse the user.

I added a comment so I can address this in a subsequent PR.